app/models/pull-request.js

```diff
@@ -16,7 +17,6 @@ export default class GithubPull extends GithubPullModel {
 
   // Links
   @attr('string') htmlUrl;
-  @attr('string') repositoryUrl;
```

Comment:
`repository_url` is a key that exists in [one endpoint](https://docs.github.com/en/rest/reference/search#search-issues-and-pull-requests) but [not the other](https://docs.github.com/en/rest/reference/pulls#list-pull-requests) that we have been using.